redux/slices/userSlice/userThunk.ts

```diff
@@ -8,15 +8,40 @@ interface FetchUserType {
   _id: string;
   doneTasks: NewUserDoneTasksDataType;
 }
+interface FinishUserTaskType {
+  taskId: string;
+  category: string;
+  taskImportance: string;
+  doneTasks: NewUserDoneTasksDataType;
+}
+
+export const fetchUserDetails = createAsyncThunk<
+  FetchUserType,
+  void,
+  { rejectValue: string }

```

Comment:
To do osobnego typu który będzie dostępnie globalnie, bo pewnie będzie często używany.